purchase_invoice_plan/wizard/create_invoice.py

```diff
@@ -18,7 +18,4 @@ def create_purchase_invoices(self):
                 for order in purchase_orders:
                     inv_ids = order.action_invoice_create()
                     invoice_ids.append(inv_ids)
-        res = self.env.ref('account.action_invoice_tree2')
-        res = res.read()[0]
-        res['domain'] = "[('id','in',["+','.join(map(str, invoice_ids))+"])]"
-        return res
+        return True
```

Comment:
If there are > 1, i.e., 3 installment, it will redirect to show only 1 invoices (while all 3 is created).
But, I think it is OK to not redirect to invoice tree view. User will just click on Invoice smart button to redirect. (It is also ok as for big organization, the purchaser may not need to see the invoices).